modules/swagger-codegen/src/main/java/io/swagger/codegen/CodegenParameter.java

```diff
@@ -122,7 +122,9 @@ public CodegenParameter copy() {
         if (this.items != null) {
             output.items = this.items;
         }
-        output.vendorExtensions = this.vendorExtensions;
+        if(this.vendorExtensions != null){
+        	output.vendorExtensions = new HashMap<String, Object>(this.vendorExtensions);
```

Comment:
Please use 4-space instead of tab.

Also what about `vendorExtensions` in other objects (e.g. CodegenProperty) ?